.github/workflows/ci.yml

```
@@ -12,6 +12,19 @@ on:
   workflow_dispatch:
 
 jobs:
+  label:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)